addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.js

```diff
@@ -8,14 +9,24 @@ export class AvatarCardPopover extends Component {
     static props = {
         id: { type: Number, required: true },
         close: { type: Function, required: true },
+        relation: { type: String, optional: true },
+    };
+    static defaultProps = {
+        relation: "res.users",
     };
 
     setup() {
         this.actionService = useService("action");
-        this.orm = useService("orm");
         this.openChat = useOpenChat("res.users");
         onWillStart(async () => {
-            [this.user] = await this.orm.read("res.users", [this.props.id], this.fieldNames);
+            this.user = await rpc("/discuss/avatar_card", {
```

Comment:
This would crash if doing code pull but no server restart.
I guess it's acceptable on saas-versions as they do server restart when they pull.

In major versions we should be more careful and only call the new route if the server is restarted, which can only be checked if the server is sending a new key at init or something like that.
Or checking if the rpc works and handling the error.